modules/ROOT/pages/6.0-release-notes-known-issues.adoc

```diff
@@ -23,4 +23,13 @@ As of this release, however, the specific cause is still being investigated.
 Workaround::
 Other than keeping `nonbreaking_force_tab` set to `false`, no workaround is currently available.
 
-// end::known-issues[]
\ No newline at end of file
+== Editor content can overlap with an open floating toolbar drawer when using certain premium skins
+
+In {productname} 6.0, if one of the premium skins, `snow`, `naked`, `outside`, or `fabric` is used and the floating toolbar drawer is open, content within the editor may visually overlap the toolbar drawer.
```

Comment:
```suggestion
In {productname} 6.0, if

* the floating toolbar drawer is open; and

* one of the following premium skins is being used

  — `snow`, `naked`, `outside`, or `fabric` —

content within the {productname} editor may overlap visually with the toolbar drawer.
```

Reply:
Inferring the problem from ltrouton’s note about not being _super confident about the wording_.

The above re-write is an attempt to use visual presentation (eg a bulleted list and em-dashes) to effect how the text is read. Which, I further hope, makes the complexity of the problem being documented clearer to said reader.

Reply:
Thanks for the suggestion @brian-forte. I have just updated it slightly to render correctly

```suggestion
In {productname} 6.0, if

* the floating toolbar drawer is open; and
* one of the following premium skins is being used
+
— `snow`, `naked`, `outside`, or `fabric` —

content within the {productname} editor may overlap visually with the toolbar drawer.
```

Reply:
This still looks to be rendering in a way that's confusing to me at least:
![image](https://user-images.githubusercontent.com/1575550/167516312-31011f8c-c166-4b9a-beba-f6b7d4225007.png)

I really think we need the names on the same line as the list item.